### PR TITLE
Support of Fashion MNIST and GTS datasets

### PR DESCRIPTION
**Pull request by max-andr**
Hi Vincent,

I have added 2 more datasets: Fashion MNIST (straightforwardly from keras.datasets) and GTS (http://benchmark.ini.rub.de/?section=gtsrb&subsection=dataset) from my .mat files. Note that GTS originally has images of different sizes, but I have cropped them all to 32x32 to make this dataset comparable to CIFAR-10 in terms of the input dimensions.

It's probably obvious, but one has to enable the new datasets in MIPVerify.jl package by substituting https://github.com/vtjeng/MIPVerify.jl/blob/master/src/utils/import_datasets.jl#L96 with:
`if name in ["mnist", "cifar10", "fmnist", "gts"]`
And also to change the corresponding docstring there.

I hope this might be useful for others.

### CODE REVIEW

**Comment by vtjeng** [comment]
Thank you for your contribution! I'll merge this in when I've had the time to add the change to `MIPVerify`. (If you have time to open a PR there, that would be amazing).

A few questions:
  + Do you have the script used to generate `gts_int_test.mat` and `gts_int_train.mat` available? I want to make sure that the `.mat` can be re-generated via the scripts (as they are for the Fashion MNIST dataset).
  + Do you have any example neural nets on hand (in the `.mat` form that the `MIPVerify` package expects) for the task of Fashion MNIST and GTS classification? For each dataset, I'd like to have a (simple) net that achieves a reasonable amount of accuracy as a sanity check that we imported the dataset right.

**Comment by vtjeng** [comment]
One other question: in the branch, you had a second commit that added a singleton dimension, but the commit didn't have a corresponding update to the script. Is there any chance that your change was something like this?

https://github.com/AdamHillier/MIPVerify_data/blob/10502631c4cb605d6877d65e52730f5814393312/datasets/fashion_mnist/generate_fashion_mnist_datafiles.py#L17-L19

```
(x_train, y_train), (x_test, y_test) = fashion_mnist.load_data()

# pad final dimension of dataset
x_train_f = np.expand_dims(x_train, 3)
x_test_f = np.expand_dims(x_test, 3)

sio.savemat('fashion_mnist_int_train.mat', {'images': x_train_f, 'labels': y_train})
sio.savemat('fashion_mnist_int_test.mat', {'images': x_test_f, 'labels': y_test})
```

**Comment by max-andr** [comment]
Yes, indeed, I did it by adding a singleton dimension as the last (3rd) dimension as in your code snippet.

**Comment by max-andr** [comment]
> Do you have the script used to generate gts_int_test.mat and gts_int_train.mat available? I want to make sure that the .mat can be re-generated via the scripts (as they are for the Fashion MNIST dataset).

In some form, the scripts for GTS can be found here (based on my pull request to the robust-ml library):
https://github.com/robust-ml/robustml/blob/master/robustml/provider.py


> Do you have any example neural nets on hand (in the .mat form that the MIPVerify package expects) for the task of Fashion MNIST and GTS classification? For each dataset, I'd like to have a (simple) net that achieves a reasonable amount of accuracy as a sanity check that we imported the dataset right.

I have models from our [AISTATS paper](https://arxiv.org/abs/1810.07481) in .mat format on FMNIST and GTS. For example, standardly trained models are here:
https://github.com/max-andr/provable-robustness-max-linear-regions/tree/master/models/plain
However, as far as I remember, their format differs a bit from the one you are using in your library.

**Comment by vtjeng** [comment]
This PR was closed as a result of me cleaning the repository history to avoid unnecessary large files. I'll try to merge in the changes as soon as I can.
